test-js.js

```diff
@@ -1,34 +0,0 @@
-const promisify = require('util').promisify ||
```

Comment:
Since we're introducing breaking changes anyway with the exported members, I figured this smoke test doesn't hold a lot of value any more. Correct me if I'm wrong.